src/Ledger/Ratify.lagda

```diff
@@ -249,7 +427,19 @@ delayingAction Info                 = false
 delayed : (a : GovAction) → NeedsHash a → EnactState → Bool → Set
 delayed a h es d = ¬ verifyPrev a h es ⊎ d ≡ true
 \end{code}
+} %% end small
+\caption{%Ratify iv:
+Determination of the status of ratification of the governance action}
+\label{fig:defs:ratify-iv}
 \end{figure*}
+The code in Figure~\ref{fig:defs:ratify-iv} defines still more types required for ratification of a governance action.
+\begin{itemize}
+  \item \verifyPrev takes the given governance action, its \NeedsHash, and \EnactState record and checks whether
```

Comment:
```suggestion
  \item \verifyPrev takes a governance action, its \NeedsHash, and \EnactState and checks whether the ratification restrictions are met.
```

Or something similar, linking to the section is probably a good idea. Alternatively, `verifyPrev` could be moved into that section.

Reply:
Oh, is this itemized list not in the same section as the Agda definition of `verifyPrev`?  Hopefully that will be resolved by the change from `[h]` to `[h!]` mentioned above.